cvat/apps/documentation/user_guide.md

```diff
@@ -1564,70 +1557,86 @@ Shapes that don't have `group_id`, will be highlighted in white.
-| `height`   | number of px or `width`                                | shape height                                                                                               |
-| `label`    | `"text"` or `["text"]`                                 | label name                                                                                                 |
-| `serverID` | number                                                 | ID of the object on server <br> (You can find out by forming a link to the object through the Action menu) |
-| `clientID` | number                                                 | ID of the object in your client (indicated on the objects sidebar)                                         |
-| `type`     | `"shape"`, `"track"`, `"tag"`                          | type of object                                                                                             |
-| `shape`    | `"rectangle"`,`"polygon"`, <br>`"polyline"`,`"points"` | type of shape                                                                                              |
-| `occluded` | `true` or `false`                                      | occluded properties                                                                                        |
-| `attr`     | `"text"`                                               | attribute name                                                                                             |
+| Properties  | Supported values                                       | Description                                 |
+| ----------- | ------------------------------------------------------ | --------------------------------------------|
+| `Label`     | all the label names that are in the task               | label name                                  |
+| `Type`      | shape, track or tag                                    | type of object                              |
+| `Shape`     | all shape types                                        | type of shape                               |
+| `Occluded`  | true or false                                          | occluded ([read more](#shape-mode-advanced))|
+| `Width`     | number of px or field                                  | shape width                                 |
+| `Height`    | number of px or field                                  | shape height                                |
+| `ServerID`  | number or field                                        | ID of the object on the server <br>(You can find out by forming a link to the object through the Action menu)|
+| `ObjectID`   | number or field                                        | ID of the object in your client <br>(indicated on the objects sidebar)|
+| `Attributes`| field                                                  | attribute name                              |
```

Comment:
For attributes:
Supported values: some other fields including attributes with a similar type or a specific attribute value
Description: any fields specified by a label